nbs/41_tabular.data.ipynb

```diff
@@ -581,6 +581,32 @@
     "                                  y_names=\"salary\", valid_idx=list(range(800,1000)), bs=64)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "<h4 id=\"TabularDataLoaders.test_dl\" class=\"doc_header\"><code>TabularDataLoaders.test_dl</code><a href=\"__main__.py#L31\" class=\"source_link\" style=\"float:right\">[source]</a></h4>\n",
+       "\n",
+       "> <code>TabularDataLoaders.test_dl</code>(**`test_items`**, **`rm_type_tfms`**=*`None`*, **`process`**:`bool`=*`True`*, **`inplace`**:`bool`=*`False`*, **`bs`**=*`16`*, **`shuffle`**=*`False`*, **`after_batch`**=*`None`*, **`num_workers`**=*`0`*, **`verbose`**=*`False`*, **`do_setup`**=*`True`*, **`pin_memory`**=*`False`*, **`timeout`**=*`0`*, **`batch_size`**=*`None`*, **`drop_last`**=*`False`*, **`indexed`**=*`None`*, **`n`**=*`None`*, **`device`**=*`None`*, **`persistent_workers`**=*`False`*, **`wif`**=*`None`*, **`before_iter`**=*`None`*, **`after_item`**=*`None`*, **`before_batch`**=*`None`*, **`after_iter`**=*`None`*, **`create_batches`**=*`None`*, **`create_item`**=*`None`*, **`create_batch`**=*`None`*, **`retain`**=*`None`*, **`get_idxs`**=*`None`*, **`sample`**=*`None`*, **`shuffle_fn`**=*`None`*, **`do_batch`**=*`None`*)\n",
+       "\n",
+       "Create Test [`TabDataLoader`](/tabular.core.html#TabDataLoader) from `test_items` using validation `procs`"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "show_doc(TabularDataLoaders.test_dl)"
```

Comment:
Added document for `TabularDataLoaders.test_dl` as it was missing in base.

Reply:
We shouldn't need to do this, but I'm not sure why it isn't showing up when the docs are generated.  Still hoping somebody else has an idea on why this one is having issues generating

Reply:
Actually, in the base NB,  `show_doc` is present for the following 2 methods before they have been used in example ->
- `show_doc(TabularDataLoaders.from_df)`
- `show_doc(TabularDataLoaders.from_csv)`

So i have added the following missing method so that it will be consistent (all 3 methods are present w/ show_doc now)->
- `show_doc(TabularDataLoaders.test_dl)`